### DISCUSSION TITLE
Basic suricat usage

### DISCUSSION THREAD

**Discussion by o-smirnov** [ratt-ru/suricat-beams#1] [Show and tell]
# Example 1: extracting beam gain towards a source

```python
from suricat.beamplots import *
from matplotlib.pyplot import *

freqs = ['856MHz', '1284MHz', '1711MHz']
beams = collect_beam_gain_to_source('beam/beam-L.bds.zarr/',
                          'obs-l2-qc2/im4htc-cad1/cubes/cube-1-120s.zarr/', 
                          '301.87632d -20.59687d',
                          freq=freqs)

for label, bg in beams.items():
  for i, f in enumerate(freqs):
    plot(bg[i], label=f)
    xlabel("timeslot")
    ylabel(label)
    legend()
    title(f"Stokes {label}")
    show()
```

**Comment by Athanaseus** [comment]
Hi @o-smirnov,

Is there a specific branch to which these instructions apply?

Additionally, I had to modify the import statement from `from suricat.beamplots import *` to `from suricat.beams import *`.

I also looked for the `collect_beam_gain_to_source` function but couldn't find it.
Please advise if I'm missing something.

Regards,
